ItemTypeOrderedSet.cs

```diff
@@ -76,33 +79,31 @@ public bool RemoveAt(int index)
 
 		public void Save(TagCompound c)
 		{
-			c.Add(_name + Suffix, _items.Select(x => x.type).ToList());
+			c.Add(_name + Suffix3, _items.Select(x => new ItemDefinition(x.type)).Concat(_unloadedItems).ToList());
```

Comment:
Instead of concatenating you can just store it separately

Reply:
However, whether a item has been unloaded is not a fixed attribute and may change each time it is loaded. Even if stored separately, it still needs to be checked every time it is loaded. I don't think it brings any benefit.

Reply:
And existing code does the same thing.

Reply:
Good point. Let's do the concat